src/dioptra/restapi/v1/models/service.py

```diff
@@ -550,14 +571,23 @@ def create(
 
         ml_model.resource.children.append(new_version.resource)
         db.session.add(new_version)
+        db.session.flush()
 
-        if commit:
-            db.session.commit()
-            log.debug(
-                "Model registration successful",
-                model_id=ml_model.resource_id,
-                name=ml_model.name,
+        mlflow_model_name = f"resource_{ml_model.resource_id:09d}"
```

Comment:
I am also wondering whether it's better to just generate a random UUID for each model name, this feels brittle because its a naming convention as opposed to an explicit declaration in the database.